bin/processGsRawData.sh

```diff
@@ -883,7 +887,7 @@ else
 		#       proper prm mount on the GD clusters and this script can run a GD cluster
 		#       instead of on a research cluster.
 		#
-		if [[ -e "${JOB_CONTROLE_FILE_BASE}.finished" ]]
+		if [[ -e "${JOB_CONTROLE_FILE_BASE}.GSfinished" ]]
```

Comment:
Was dit nu nog nodig? Ik dacht het niet, omdat we alleen nog mailen voor status failed. En geen T&T meer doen voor status. Voor T&T doen we nu alleen upload van de samplesheet met de config in ```etc/umcg-genomescan.cfg```, toch?